src/structures/ThreadChannel.js

```diff
@@ -439,7 +445,12 @@ class ThreadChannel extends Channel {
    * @readonly
    */
   get manageable() {
-    return this.permissionsFor(this.client.user)?.has(Permissions.FLAGS.MANAGE_THREADS, false);
+    const permissions = this.permissionsFor(this.client.user);
+    if (!permissions) return false;
+    // This flag allows managing even if timed out
+    if (permissions.has(Permissions.FLAGS.ADMINISTRATOR, false)) return true;
+    if (!permissions.has(Permissions.FLAGS.MANAGE_THREADS, false)) return false;
+    return this.guild.me.communicationDisabledUntilTimestamp < Date.now();
```

Comment:
Can't we use a single return for `MANAGE_THREADS` and `timeouts` here?
```suggestion
    return this.guild.me.communicationDisabledUntilTimestamp < Date.now() && permissions.has(Permissions.FLAGS.MANAGE_THREADS, false);
```